src/metrics.cc

```diff
@@ -109,6 +109,11 @@ Metrics::Metrics()
                     "execution per-model.")
               .Register(*registry_)),
 
+      model_load_time_family_(prometheus::BuildGauge()
+                                  .Name("nv_model_load_time")
+                                  .Help("Load Time per-model in seconds")
```

Comment:
How about "Model loading duration in seconds"?
I don't think we need to explicitly say per-model as the metric label already tells.